torch/testing/_internal/common_methods_invocations.py

```diff
@@ -2700,6 +2700,46 @@ def sample_inputs_fliplr_flipud(op_info, device, dtype, requires_grad, **kwargs)
     )
     return [SampleInput(tensor) for tensor in tensors]
 
+def sample_inputs_fmod_remainder(op_info, device, dtype, requires_grad, *, autodiffed_args=False, **kwargs):
+    make_arg = partial(make_tensor, dtype=dtype, device=device, requires_grad=requires_grad)
+
+    if autodiffed_args:
+        samples = (
+            ((S, S, S), 1.5, False),
+            ((), 1.5, False),
+        )
+    else:
+        cases = (
+            ((S, S, S), (), False),
+            ((S, S, S), (S, S, S), False),
+            ((S, S, S), (S,), False),
+        )
+
+        # Sample inputs with scalars as torch tensors
+        cases_with_tensor_scalar = (
+            ((), torch.tensor(1, dtype=dtype, device=device, requires_grad=False), False),
```

Comment:
If using `1.5` instead of `1.0`, following `DeprecationWarning` is raised. I guess to solve the purpose of testing, even using 1 should work, but is there a better way to create a scalar tensor for all the dtypes without any such warning raised?

```
DeprecationWarning: an integer is required (got type float).  Implicit conversion to integers using __int__ is deprecated, and may be removed in a future version of Python.
```